### PR TITLE
Improve the walkthrough image style

### PR DESCRIPTION
**Pull request by maxagin**
When the info looks more professional we have better chances to have the user's focus.

https://user-images.githubusercontent.com/98249521/196802745-1f8fe75f-0995-4d4a-8d5e-c05df69f865d.mov

https://github.com/iterative/design/issues/42

### CODE REVIEW

**Review by mattseddon — CHANGES_REQUESTED** [review]
You have deleted some files that are being used in the walkthrough. E.g `extension/resources/walkthrough/images/plots-dump-image.png`:

image

Please check that all of the walkthrough is rendered as expected.

**Comment by maxagin** [comment]
Sorry @mattseddon , I thought the images are replaced with the code snippets. Will create an update now.

**Comment by mattseddon** [comment]
@maxagin all of the experiments table screenshots are out of date already.

**Comment by maxagin** [comment]
> @maxagin all of the experiments table screenshots are out of date already.

@mattseddon do you suggest replacing all the old screenshots?

**Comment by mattseddon** [comment]
> > @maxagin all of the experiments table screenshots are out of date already.
> 
> @mattseddon do you suggest replacing all the old screenshots?

If you are going to take the time to change them then they should be correct.

File: extension/resources/walkthrough/plots.md [hunk 1]
```diff
@@ -1,9 +1,17 @@
 # Plots Dashboard
 
-Select one or more experiments to visualize in the
-[**Plots Dashboard**](command:dvc.showPlots). Use
+Select (click the circle beside the experiment name) up to 7 experiments in the
+table view to visualize in the [**Plots Dashboard**](command:dvc.showPlots).
+
+<p align="center">
+  <img src="images/plots-click-the-circle-beside-experiment-name.png"
```
Comment (mattseddon) [inline extension/resources/walkthrough/plots.md:7]: This one is still missing: 

image

**Review by mattseddon — DISMISSED** [review]
Please check that all images work and record a demo for the description

**Comment by mattseddon** [comment]
> When the info looks more professional we have better chances to have the user's focus.
> 
>  Screen.Recording.2022-10-18.at.11.33.50.PM.mov 
> [iterative/design#42](https://github.com/iterative/design/issues/42)

@maxagin the demo should be from inside the product.

**Comment by shcheklein** [comment]
@mattseddon @maxagin what is left here folks?

**Comment by mattseddon** [comment]
@shcheklein resolving the conflict isn't worth the effort as this branch pre-dates the demo project being moved into a submodule. I'll lift the images from this branch into a new one and open a new PR.

**Pull request closed without merging**